build.gradle

```diff
@@ -1,13 +1,13 @@
 buildscript {
   repositories { jcenter() }
-  dependencies { classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:2.+' }
+  dependencies { classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:2.2.3' }
```

Comment:
Lots of little things in this PR, I might separate the ```build.gradle``` change to another PR for separate consideration.